### ISSUE TITLE
Accuracy and pre-trained models

### ISSUE THREAD

**Issue by idanh** [adamzjk/Age-Gender-Pred#2]
Hi, super nice project!
I was wondering if you evaluated the model with mae?
And if possible, would you mind sharing your pretrained model, maybe as a release?
Thanks!

**Comment by SantoshBajantri** [comment]
> Hi, super nice project!
> I was wondering if you evaluated the model with mae?
> And if possible, would you mind sharing your pretrained model, maybe as a release?
> Thanks!

Hey did you get pre-trained model? It would be helpful if you could send the model. I'm facing some problems in training the model.
Thank You

**Comment by idanh** [comment]
@SantoshBajantri  I did not. Nobody answered, as you can see..
And I ended up using: https://github.com/yu4u/age-gender-estimation
Which was more active at the time..

**Comment by SantoshBajantri** [comment]
@idanh Ok thanks for the reply. I have already tried the one which you have shared, but it lacks in accuracy. Please suggest any other which performance with at-least 90% accuracy. Or share the model if you have trained your own dataset here: https://github.com/yu4u/age-gender-estimation

**Comment by SandeepSreenivasan** [comment]
@idanh  @SantoshBajantri 

I tried the pretrained model from https://github.com/yu4u/age-gender-estimation, as @SantoshBajantri has mentioned, the accuracy is bit less. Did you people try out training the model with more dataset, did you find any model with better accuracy? Could you give me some heads up regarding how to go along with.

**Comment by veeri** [comment]
i am using imdb images for training, as recommended in this project docs, when i run preprocess.py file it will generate the labeled folder with images, file is like age_gender_person_name, but for some images gender and age is wrong you can see in your labeled folder, with those images accuracy giving less, or is there any other issue about accuracy , can you correct me

**Comment by maria1-mln** [comment]
> i am using imdb images for training, as recommended in this project docs, when i run preprocess.py file it will generate the labeled folder with images, file is like age_gender_person_name, but for some images gender and age is wrong you can see in your labeled folder, with those images accuracy giving less, or is there any other issue about accuracy , can you correct me
I also encountered the same problem, did you solve your problem？
